lib/tasks/dev.rake

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate test data'
+  task :generate_data, [:dev_num] => :environment do |_, args|
+    get_args(args)
+
+    create_programming_language
+    create_language
+
+    programming_languages = ProgrammingLanguage.all
+    languages = Language.all
+
+    @args[:dev_num].times do
+      FactoryBot.create :developer,
+                        programming_languages: programming_languages.sample(rand(1..4)),
+                        languages: languages.sample(rand(1..3))
+    end
+  end
+end
+
+def create_programming_language
```

Comment:
The method name should be plural, `create_programming_languages`, because you create multiple records.

Reply:
Fixed!